P5/Source/Specs/noteGrp.xml

```diff
@@ -0,0 +1,138 @@
+          <att>n</att>-Attribut benutzt.</p>
+      </remarks>
+    </attDef>
+    <attDef ident="targetEnd" usage="opt">
+      <gloss versionDate="2017-06-25" xml:lang="de">Bereichsende</gloss>
+      <desc versionDate="2005-01-14" xml:lang="en">points to the end of the span to which the note is attached, if the note is not embedded
+        in the text at that point.</desc>
+      <desc versionDate="2009-01-06" xml:lang="fr">pointe vers la fin d'un passage auquel la note est
+        attachée, si la note n'est pas enchâssée dans le texte à cet endroit</desc>
+      <desc versionDate="2007-05-04" xml:lang="es">indica el final de un periodo al cual una nota está
+        enlazada, si la nota no está insertada en ese punto del texto.</desc>
+      <desc versionDate="2007-05-02" xml:lang="zh-TW">如果附註不是嵌在正文中，則用以標明附註所註段落的終點。</desc>
+      <desc versionDate="2007-01-21" xml:lang="it">rimanda alla fine della porzione di testo a cui la
+        nota è allegata nel caso in cui la nota non sia inclusa nel testo in quel punto.</desc>
+      <desc versionDate="2006-10-28" xml:lang="ja">当該注釈・コメントの終了点を示す．(当該注釈が注釈先の場所に埋め込 まれていない場合)</desc>
+      <desc versionDate="2017-06-25" xml:lang="de">verweist auf das Ende eines Bereichs, dem das <gi>note</gi>-Element angefügt ist, es sei denn die
+        Anmerkung ist in den Text an diesem Punkt bereits eingebettet.</desc>
+      <datatype maxOccurs="unbounded"><dataRef key="teidata.pointer"/></datatype>
+      <remarks versionDate="2009-01-23" xml:lang="en">
+        <p>This attribute is retained for backwards compatibility; it may be removed at a subsequent
+          release of the Guidelines. The recommended way of pointing to a span of elements is by
+          means of the <ident>range</ident> function of XPointer, as further described in <ptr target="#SATSRN"/>. </p>
+      </remarks>
+      <remarks xml:lang="fr" versionDate="2007-06-12">
+        <p>Cet attribut est conservé pour assurer un arrière-plan compatible ; il sera supprimé dans la
+          prochaine mise à jour des Recommandations. La procédure recommandée pour pointer en
+          direction d'une expansion des éléments est de le faire au moyen de la fonction
+            <ident>range</ident> de XPointer, telle que la description en est faite  à <ptr target="#SATSRN"/>. </p>
+      </remarks>
+      <remarks versionDate="2017-06-25" xml:lang="de">
+        <p>Dieses Attribut wurde beibehalten, um eine Abwärtskompatibilität zu gewährleisten; bei einem
+          zukünftigen Release der Guidelines kann es entfernt werden. Der empfohlene Weg, um auf einen
+          Bereich zu verweisen, ist die Verwendung der <ident>range</ident>-Funktion von XPointer, wie in
+          <ptr target="#SATSRN"/> beschrieben.</p>
+      </remarks>
+    </attDef>
+  </attList>
+  <exemplum xml:lang="la">
+    <p>In the following example, two notes specifying the content of the annotation relating to the
```

Comment:
Confusing sentence. How about “In the following example, there are two notes in different languages, each specifying the content of the annotation relating to the preceding fragment of text.”?